default startup.py location hint (#3479)
qgis/QGIS-Documentation: source/docs/pyqgis_developer_cookbook/intro.rst

is searched for a file named :file:`startup.py`. If that file exists, it
is executed by the embedded Python interpreter.

> .. note:: The default path can differ on each system. To find out, which path will work for you, open Python Console and run `QStandardPaths.standardLocations( QStandardPaths.AppDataLocation )` to see list of default Python directories.

Reviewer: ```suggestion
.. note:: The default path depends on the operating system.
   To find the path that will work for you, open the Python Console and run
    `QStandardPaths.standardLocations(QStandardPaths.AppDataLocation )` to see the list
    of default directories.
```

> 

Reviewer 2: Thanks! They are not "Python" directories btw, they are "standard" directories according to QT
